ci/Jenkinsfile-kubernetes

```diff
@@ -459,10 +481,11 @@ def testScalaExternalBackendAutoClientMode(registryId, master, version) {
     sh "kubectl delete pod sparkling-water-app --ignore-not-found"
     installSparkHeadlessService()
     def image = "${registryId}.dkr.ecr.${awsRegion()}.amazonaws.com/sw_kubernetes_repo/sparkling-water:scala-${version}"
-    sh """
+    runCommandWithK8sDebugInfo"""
         kubectl run -n default -i --tty sparkling-water-app --restart=Never --labels spark-driver-selector=sparkling-water-app \
         --image=${image} -- ${scalaExternalBackendSubmitCmd(registryId, master, version, "client", "auto")}
         """
+    sh "sleep 330"
```

Comment:
In automatic external backend, the driver pod still runs 5 minutes after everything is finished. This needs to be fixed during the re-work to H2O K8s operator.

Reply:
Sure, this is rather a minor issue given the overall state of the test pipeline. Would duplicate efforts. Makes all sense to just ignore ATM.